src/Equinox.CosmosStore/CosmosStore.fs

```diff
@@ -394,8 +394,7 @@ module private MicrosoftAzureCosmosWrappers =
     type Container with
         member container.TryReadItem(partitionKey : PartitionKey, documentId : string, ?options : ItemRequestOptions): Async<float * ReadResult<'T>> = async {
             let! ct = Async.CancellationToken
-            let options = defaultArg options null
-            use! rm = async { return! container.ReadItemStreamAsync(documentId, partitionKey, requestOptions = options, cancellationToken = ct) |> Async.AwaitTaskCorrect }
+            use! rm = async { return! container.ReadItemStreamAsync(documentId, partitionKey, requestOptions = Option.toObj options, cancellationToken = ct) |> Async.AwaitTaskCorrect }
```

Comment:
Do we need this `async` block at all?

Reply:
Good point - Struggling to think of the remotest justification so applied it. Same construct is in V2 - I think the original code was grabbed from some generic helpers somewhere!